taipy/gui/_renderers/factory.py

```diff
@@ -444,6 +444,24 @@ class _Factory:
                 ("width", PropertyType.string_or_number),
             ]
         ),
+        "progress": lambda gui, control_type, attrs: _Builder(
+            gui=gui,
+            control_type=control_type,
+            element_name="Progress",
+            attributes=attrs,
+        )
+        .set_value_and_default(var_type=PropertyType.dynamic_number, native_type=True)
+        .set_attributes(
+            [
+                ("linear", PropertyType.boolean, False),
+                ("show_value", PropertyType.boolean, False),
+                ("title", PropertyType.dynamic_string),
+                ("title_anchor", PropertyType.string, "bottom"),
+                ("render", PropertyType.dynamic_boolean, True),
+                ("width", PropertyType.string_or_number),
+            ]
+        )
+        ._set_propagate(),
```

Comment:
don't think `propagate` is needed here as there's no variable modification from the frontend

Reply:
Let's discuss this rapidly. I'm sure you're right.

Reply:
Done.